app/components/App.jsx

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
+import WidgetHeader from './WidgetHeader';
+import WidgetMessage from './WidgetMessage';
+
+const SERVER_DATA = {
```

Comment:
You're right it should go in a different store, but I'm faking it for now to have it running, and I'll deal with server calls later.